### DISCUSSION TITLE
What is the criteria for Abandoned?

### DISCUSSION THREAD

**Discussion by aucampia** [veggiemonk/awesome-docker#1026] [Q&A] [answered]
What would you consider abandoned? 1 year with no releases? last closed issue more than 1 year ago? If you give some indication I will make a PR to clarify it in your list.

**Comment by mashb1t** [comment, marked as answer]
Hey @aucampia,

i am currently not aware of strict guidelines on when a repository or project on the list will be marked as abandoned, but i assume it's

* in the range of 2-3 years without releases
* if the project doesn't get patched for critical fixes for a while
* if the project maintainer mentiones that it's abandoned in the corresponding readme
* if somebody creates a PR for marking it abandoned

We do a irregular but mostly yearly check of projects to identify non-patched ones and update this list accordingly.

You're welcome to create a PR for marking repositories as abandoned if you find hints on their status.
